src/components/TestnetNotification.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import styled from 'styled-components';
+import { Alert } from './Alert';
+
+const StyledAlert = styled(Alert)`
+  text-align: center;
+`;
+
+export const TestnetNotification = (): JSX.Element => {
+  return (
+    <StyledAlert variant="warning" round="none" pad="small">
+      <FormattedMessage defaultMessage="Warning: This is a staking launchpad for TESTNET." />
```

Comment:
Slight wording suggestion:
```suggestion
      <FormattedMessage defaultMessage="Warning: This is a staking launchpad for the TESTNET." />
```

Or, we could pull the all-caps styling into css so it doesn't get goofed up in translation:
```suggestion
      <FormattedMessage
        defaultMessage="Warning: This is a staking launchpad for {testnet}."
        values={{ testnet: <AllCaps><FormattedMessage defaultMessage="testnet" /></AllCaps> }}
      />
```
(Plus the `const AllCaps` suggestion above)